librz/core/cmeta.c

```diff
@@ -363,3 +363,128 @@ RZ_IPI void rz_core_meta_editor(RzCore *core, RzAnalysisMetaType mtype, ut64 add
 		free(out);
 	}
 }
+
+/**
+ * RzCoreMetaString
+ * @{
```

Comment:
?

Reply:
It's a grouping Doxygen feature.

Reply:
https://www.doxygen.nl/manual/grouping.html

Reply:
woah, ok :D